### ISSUE TITLE
Change colour used in sortie node indicator

### ISSUE THREAD

**Issue by Accelyte** [KC3Kai/KC3Kai#2168]
Red on red is hard to read when the icon is so small. Wikia uses a yellow circle for the air raid, so maybe it can be changed?

**Comment by sinsinpub** [comment]
how about Night Battle node, Aerial Battle node, and Enemy Fleets Combined node?

**Comment by sinsinpub** [comment]
![image](https://user-images.githubusercontent.com/160240/29360913-ab3e6b28-82b7-11e7-86b1-1ed0d9dabf9e.png)
A: combined, C: night starts, F: aerial battle, J: long distance air raid

they are drawn with my poor css `background: linear-gradien` knowledge. open for any color suggestion, but please consider the balance of all cases. give me css codes if suggests on shape/other things 😃

**Comment by Accelyte** [comment]
I was just using air raid node as an example.
Here's some suggestions (I don't know much about css, so no css codes here...I can doodle in illustrator tho, I suppose you could implement complex vectors like the arrows as a background image to an imgur link?)

Long Distance Air Raid: Yellow arrow like how kancolle indicates it, something like I scratched out here...
![air-raid](https://user-images.githubusercontent.com/21363469/29366658-2d9d3242-82cd-11e7-8d82-da78d69626a1.png)

Night Battle: Blue circle
![scratch2](https://user-images.githubusercontent.com/21363469/29366803-9e8d3dee-82cd-11e7-8e2a-42be9d51aae2.png)

Day into Night Battle: red/blue half: red on left, blue on right, since day comes first (a little bit of rotation so if it's node I, the letter doesn't get blocked)
![dn](https://user-images.githubusercontent.com/21363469/29366909-d808cc78-82cd-11e7-89aa-d0f73fedb1b3.png)

Combined Fleet: two green straight lines, since combined fleet is two rows of ships
![scratch](https://user-images.githubusercontent.com/21363469/29366168-e0022412-82cb-11e7-8623-e54ff0515d77.png)

I forgot what a normal aerial battle is so I didn't doodle it. Also, maybe arrows are too small to draw?

**Comment by nonsensed** [comment]
I'm pretty far into the event and therefore my sanity is pretty low. I don't want to make any suggestions without being pretty sure my suggestions would help and low sanity is not good for this.

Still, I want to say be careful with what colors are picked. The node letter is white, so bright colors will make the letter hard to see. A lot of people have reacted to the air raid node icon because it's the only real one in this event. For this icon, the triangle thing is a brighter color, making it slightly harder to read. A vivid red background with white text is dangerous.

**Comment by nonsensed** [comment]
I finally tried making some icons, too, but I don't know if this is any good.
![abyssalplane](http://i.imgur.com/30FaJvL.png)
![aircraft](http://i.imgur.com/sL4WgMq.png)

They could go above the node in the panel like this (not coded in btw-- only shopped):
Long-Distance Air Raid
![longdistance](http://i.imgur.com/DemERWT.png)
Aerial Battle:
![aerialbattle](http://i.imgur.com/EFUlAfw.png)

In both cases, an air raid panel circle color was used: `39B74E` . In my opinion, if the icon is different above the node letter, then it's ok to have both kinds of air battles as the same color.

and then a night battle node could colored like this (`001A3E`):
![yasen](http://i.imgur.com/1ez8JBd.png)

Combined fleet seems ok to not show a different node icon for. I think anyone doing an event will still use the same formations and everything, so notifying the player of a combined fleet icon isn't very important. It's definitely not because I didn't feel like trying to find a color(!).

I have no idea if this is a good idea or not. But if I don't write it down, I will probably be too lazy to post this later.

**Comment by Accelyte** [comment]
The plane icon idea is good, but I had to put my face right in front of the monitor to see it, particularly since I'm using a laptop. Using various shapes is probably easier on the eyes, but I haven't thought about how to apply them yet.

**Comment by nonsensed** [comment]
I actually downsized them in the node screenshots and forgot about it so the quality is poor, but yeah I guess if the icons are too hard to see, then that's not good.

**Comment by ethrundr** [comment]
IMO all the icon submitted by @Accelyte is good enough. However, there are still some things I'd like to suggest/improve:

- Combined and Air Raid icon: It's already good, but the lines and arrow need to be bit thicker, given that this icon will be downsized into a small circle. This to prevent the line becoming too thin, thus can't be seen.
- The third icon, it should be "Battle start at Night and proceed to Day (Night-to-Day --> IBS reference). So the blue and red color should be reversed: Blue first, then red.
- For Aerial Battle node (AL/MI reference): I think about a single arrow (in blue or other color), pointed to right (or above-right).

Sorry for not being able provide an example.

**Comment by sinsinpub** [comment]
haha, i just come to clarify: keep watching game screen when play game.
imo these indicators are just a thing for 'backlog' when ingame screen of map not visible. will be good enough as long as they are clear to read without ambitious. making strategy decision (such as formation selection) on it seems weird. it's enough on game screen to make the decision, besides the 'night battle node' and 'abyssal combined node' (but your formation not important for them).

for examples:
Aerial battle node (1-6-F):
![image](https://user-images.githubusercontent.com/160240/29503008-df179b10-8666-11e7-8f5e-049ef19db51c.png)

Air raid node (6-5), also with abyssal planes engaging animation:
![image](https://user-images.githubusercontent.com/160240/29503062-48f437d2-8667-11e7-8149-e207611e4f87.png)

so put the 'single or double array' icons similar with the ingame screen ones above onto node background might be the best choice, don't need to explain what are they. 🤔 
however, don't worry, we already have tool-tips on these indicators to explain.

**Comment by ethrundr** [comment]
I'm aware of that, just suggesting something that still similar but easier to be recognize without navigating for the tooltip.

**Comment by ethrundr** [comment]
Also, regarding @Accelyte had said:
> The formation selection screen blocks most of the map so you might not see which node you're on\
and while we're at this, I'd like to address one more node icons: **_How about ASW nodes?_**

The map itself indeed does not differentiate between normal battle and ASW nodes.
However, due to the different formation required for ASW nodes, giving a special icon might be better - the logic is the same like what we did on night battle node.

**Comment by nonsensed** [comment]
I think I remember sinsin saying submarine nodes couldn't be detected when he was originally starting to look into the node icon thing. It was on discord.

**Comment by sinsinpub** [comment]
yep, it's not possible to determine a node is 'ASW node' or not via `api_req_map/next` API data.

kc devs only put the submarine animation into each map swf file, so if anyone of our devs could build & update a client-side data file for every implemented maps like the `edge.json`, i'm also wishing to indicate submarine node.

and there are also exceptions for devs' swf file:
 * some nodes are forgotten to add submarine animation, such as 6-2-C. our node marker already indicate it.
 * some nodes are partially submarine only, other random patterns are surface or mixed, such as 6-2-B, 6-4-E

**Comment by ethrundr** [comment]
Ah, I see. OK then.

**Comment by Accelyte** [comment]
I'm not saying you should be relying on KC3 for decision-making, I said that if it can serve a wider purpose if implemented properly. I agree that it is a helper tool, and _proper_ implementation is a method of helping. If you're adding too much info to the implementation then that's overkill, which I never suggested.

**Comment by nonsensed** [comment]
(We're veering off-topic, but personally I don't see a problem with too much information, provided it doesn't overwhelm the user and make them TL;DR)

**Comment by Accelyte** [comment]
I just wanted to clarify my point because someone clearly got the wrong idea as to exactly what I am trying to say.

**Comment by ethrundr** [comment]
I personally also against putting too much information on the KC3, as it will render the user being too reliant with what are provided. I'm always on this stance since the first time I joined the team and will always be.

I also does not mean to offend anyone with my suggestion. I understand clearly what you are trying to say here, and I try to come up with what I think good to make it better. All of my suggestions posted for KC3, either here or (rarely) discord, are all merely based on my own experience on using the KC3, playing the game, and gathering all suggestions from the user-community (both KC and KC3) in my own group.
